composer.json

```diff
@@ -45,6 +45,8 @@
         "phpstan/phpstan": "~1.4.10 || 1.10.14",
         "phpunit/phpunit": "^7.5 || ^8.5 || ^9.6",
         "psr/log": "^1 || ^2 || ^3",
+        "ramsey/uuid": "^4.7",
+        "ramsey/uuid-doctrine": "^2.0",
```

Comment:
Other route: create a custom type in the test: https://www.doctrine-project.org/projects/doctrine-dbal/en/current/reference/types.html#custom-mapping-types

Reply:
I did that in the first place, but SQLite couldn't be bothered in memory with my assertions for some reason. After I've set up a docker MySQL instance I got further, I'll try this again later with a simplistic example. Heading out to enjoy the sun for a bit now :-)

Reply:
> SQLite couldn't be bothered in memory with my assertions for some reason

I didn't quite get that sentence, but enjoy the :sun_with_face: !

Reply:
Oh, when I ran the test suite early just with the default SQLite in memory configuration i couldn't get a failing assertion to my surprise. But I was also asserting against what's still left over in the unit of work instead of really querying manually again, so i think that's what caused confusion.

Reply:
SQLite also failed with manual querying so I think that's where I made a wrong assumption :-)